pkg/utils/image/infos.go

```diff
@@ -51,12 +51,6 @@ func (i *ImageInfo) ReferenceWithTag() string {
 }
 
 func GetImageInfo(image string, cfg config.Configuration) (*ImageInfo, error) {
-	logger.V(3).Info(
-		"getting the image info",
-		"image", image,
-		"defaultRegistry", config.Configuration.GetDefaultRegistry(cfg),
-		"enableDefaultRegistryMutation", config.Configuration.GetEnableDefaultRegistryMutation(cfg),
```

Comment:
How about adding these two to the error [here](https://github.com/kyverno/kyverno/pull/8673/files#diff-b25f3a88d050ffd85970522b0f60cb323b8db1622ab2cf5a018ead681988e1c1R58)?